modules/samples/consumers/http-performance/src/main/java/LogService.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import com.jayway.jsonpath.JsonPath;
+import org.apache.log4j.Logger;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.text.DecimalFormat;
+import java.util.Map;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.atomic.AtomicLong;
+
+public class LogService extends HttpServlet {
+
+    private static Logger log = Logger.getLogger(LogService.class);
+
+    private DecimalFormat decimalFormat = new DecimalFormat("#");
+    private static int elapsedCount = 50000;
+    private static AtomicLong eventCount = new AtomicLong(0);
+    private static long lastTime;
+
+    private static AtomicLong latency = new AtomicLong(0);
+
+    public void init() throws ServletException {
+        System.out.println("Logger service initiated");
```

Comment:
Thanks @prabathabey , change is already done in commit https://github.com/wso2/product-cep/commit/407b21f844f766a438a6617dfaa40cf7fcf63dbc